package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Serverless containerized social arcade built on Knative.",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node index.js"
```

Comment:
Would this need to be changed to `build/app.js` from #1?

Reply:
I think so.